### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by MoritzKn**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -88,5 +88,5 @@ Our MIT license is designed to be as simple, and liberal as possible.
 [@mgreter]: https://github.com/mgreter
 [@xzyfer]: https://github.com/xzyfer
 
-sass2scss was originally written by [Marcel Greter](@mgreter)
+sass2scss was originally written by [Marcel Greter](https://github.com/mgreter)
```
Comment (xzyfer) [inline Readme.md]: The proper fix for this is to move the reference links below this line.

```
 [@hcatlin]: https://github.com/hcatlin
 [@akhleung]: https://github.com/akhleung
 [@chriseppstein]: https://github.com/chriseppstein
 [@nex3]: https://github.com/nex3
 [@mgreter]: https://github.com/mgreter
 [@xzyfer]: https://github.com/xzyfer
```

Comment (MoritzKn) [inline Readme.md]: If i move the reference links to the end of the file, the link `[Marcel Greter](@mgreter)` still don't works. It only seams to work if i write it like this`[@mgreter]`.

Comment (xzyfer) [inline Readme.md]: Correct, the `()` are a typo it should be `[]`. You may not even need to move the references.

**Comment by xzyfer** [comment]
Thanks @MoritzKn

**Pull request merged**